### ISSUE TITLE
Fully randomized seeds 1-3

### ISSUE THREAD

**Issue by p0trzebi3** [tewtal/itemrandomizerweb#69]
They all have speed booster but no bombs, so what can I do? I can't get up to the Billy M room because I can't stand up with the charge, and I fail when doing a speed machball from the ship over to the blocks in front of terminator (I fall down the hole).

Please help me, I will love you forever.

**Comment by moriwizz** [comment]
What is the seed?

**Comment by p0trzebi3** [comment]
> What is the seed?

1, 2 and 3 all have this, also 6, which is kind of strange but interesting. Speed booster in different places and no bombs/power bombs.

For example, in #1 I have morph ball, speed booster and three missile packs after kiling bomb torizo (and managing alcatraz escape after heavy training). There's got to be somewhere I'm missing where I can use the booster?

**Comment by moriwizz** [comment]
Are you saying that your seed is "1"? What is the difficulty?

**Comment by p0trzebi3** [comment]
> Are you saying that your seed is "1"? What is the difficulty?

No, that would be a two char string constant with the value 49 (assuming that you're using an ASCII based system) followed by 0.

The difficulty is mentioned in the topic.

**Comment by InsaneFirebat** [comment]
> I fail when doing a speed machball from the ship over to the blocks in front of terminator (I fall down the hole).

Try jumping over the hole, all the way into the blocks before Terminator. You'll need to release jump mid-way to control your height, but it shouldn't be too hard to pull off for a beginner to randomizer.

If you're having trouble jumping before the hole, you can do a simple short charge to get blue sooner. Start walking with the dash button released, then press and hold it after Samus takes a step forward.

**Comment by p0trzebi3** [comment]
Ah, that's what was missing. I knew about quick charge but not short charge. It's gonna make the second fly to wrecked ship easier too.

Thanks for the software. I've been having a lot of fun counting power bombs, doing short hell runs and worrying that varia suit is to the left of ridley.

The pedagogic capabilities (or interest) of speed runners to explain various tricks, could be improved, but I'll manage.

All the best.

**Comment by InsaneFirebat** [comment]
If you use Discord, I'd recommend joining the main SM speedrunning server for fast responses and multiple viewpoints. A good portion of the community is eager to share their knowledge if you know where to ask. There's also a speedrunning wiki if you prefer to research on your own. This GitHub repository isn't being actively developed, so I doubt many people are watching its comments/issues section anymore.

https://discord.gg/rT2fWZt
https://wiki.supermetroid.run
